docs/design/autotuing-design.md

```diff
@@ -0,0 +1,269 @@
+# Design to Auto-Tuning Hyperparameters with DLRover
+
+This design doc describes how DLrover auto-tunes hyperparameters a job.
```

Comment:
"hyperparameters a job." - > "hyperparameters of a job."

Reply:
Fixed by latest commit.